btrdb/stream.py

```diff
@@ -1145,6 +1151,26 @@ def windows(self, width, depth):
         specified. This is much faster to execute on the database side.
 
         """
+        start, end = None, None
+        if len(args) < 2 and len(kwargs) < 2:
+            raise UserWarning("Require 'width' and 'depth'")
+        elif len(args) == 2 and len(kwargs) == 0:
```

Comment:
Seems surprising to me that the user can specify `width` and `depth` in either the args or the kwargs. Also, it might be unclear to the user how to order `width` & `depth` when specifying as args.